Compile error occure in conn_ex_utility project

<!--
Before filing a bug
- Ensure the bug reproduces on the latest SDK.
- Search existing issues and make sure this issue is not already filed.
-->

**Describe the bug**
<!-- A clear and concise description of what the bug is. -->
A compile error C2039, and C2065 occure in conn_ex_utility project.
It seems to be a lack of header include about std::nothrow.

**To Reproduce**
<!-- Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error -->
1. clone microsoft/Azure-Kinect-Sensor-SDK
2. generate build file using CMake
3. build generated build file using Visual Studio

**Expected behavior**
<!-- A clear and concise description of what you expected to happen. -->

**Logs**
<!-- Please enable *info* level logging and attach any logs you have that your issue.
See [k4atypes.h](https://github.com/Microsoft/Azure-Kinect-Sensor-SDK/blob/feecae0456511ac734287571b101d10fd7292673/include/k4a/k4atypes.h#L184) for how to enable logs. -->

**Screenshots**
<!-- If applicable, add screenshots to help explain your problem. -->
![2021-04-02_18h28_36](https://user-images.githubusercontent.com/816705/113404122-a0869a80-93e2-11eb-95ba-c1578394c676.png)

**Desktop (please complete the following information):**
 - OS with Version: Windows 10 20H2<!-- e.g. Windows 1903 -->
 - SDK Version:  develop/HEAD<!-- e.g. 1.0.1 -->
 - Visual Studio Version: Visual Studio 2019 16.9.3
 - CMake Version: CMake 3.19.2

**Additional context**
<!-- Add any other context about the problem here. -->

--- a/tests/Utilities/ConnEx/ConnEx.cpp
+++ b/tests/Utilities/ConnEx/ConnEx.cpp
@@ -7,6 +7,7 @@
 #define WIN32_LEAN_AND_MEAN
 #include <Windows.h>
 #include <stdio.h>
+#include <new>
 
 #include <k4ainternal/logging.h>
 

